dev/README.md

```diff
@@ -0,0 +1,7 @@
+# How to setup a dev environment using VSCode and Docker
```

Comment:
What I think would be nice is if you would add a line or two explaining why Docker is useful. I guess that you simulate a dev environment, so problems like different OS versions etc are avoided?

Reply:
Done